sushichef.py

```diff
@@ -50,11 +51,87 @@ class ChickenNChipsChef(YouTubeSushiChef):
         'CHANNEL_THUMBNAIL': CHANNEL_THUMBNAIL,
         'CHANNEL_DESCRIPTION': CHANNEL_DESCRIPTION,
     }
+    DATA_DIR = os.path.abspath('chefdata')
+    DOWNLOADS_DIR = os.path.join(DATA_DIR, 'downloads')
+    ARCHIVE_DIR = os.path.join(DOWNLOADS_DIR, 'archive_{}'.format(CONTENT_ARCHIVE_VERSION))
+
+
+    def get_video_ids(self):
+        return get_video_ids(CHICKEN_N_CHIPS_CHANNEL_ID)
+
+    def get_channel_metadata(self):
+        return {
+            'defaults': {
+                'license': licenses.AllRightsLicense("Chicken&Chips"),
+                'high_resolution': True
+            }
+        }
+    
+    def sort_topic_alphabetically(self, channel):
```

Comment:
@richard-dinh Could you actually create a PR that adds this function to `TreeNode`? This seems like a generally useful thing to have.

Reply:
Maybe also make it possible to pass in a custom sort function, and use the one here if not specified.

Reply:
Just added the ability to pass in a key for the `sorted` function. Let me know what you think and if it looks good, I'll add it to `TreeNode` in `ricecooker` and create a PR